README.md

```diff
@@ -421,13 +421,18 @@ pull requests. You can find the contribution guidelines on
 
 ## Releasing new versions
 
+Updates should be added to the latest draft release on GitHub as they are made.
```

Comment:
```suggestion
Updates should be added to the latest draft release on GitHub as changes are made.
```

Maybe? or "as PRs are merged"?

Reply:
Sure, maybe "as PRs are merged" makes the instructions a little clearer 👍